### ISSUE TITLE
SORTA manual match results in error

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#5539]
#### Reproduce
- Import [biobank_ontology.owl.zip](https://github.com/molgenis/molgenis/files/591833/biobank_ontology.owl.zip) using the 'Upload' plugin
- Select the 'SORTA' plugin in the 'Data integration' menu
- Select 'New task'
- Set the name 'sorta test' for the new task
- Past the content of [sorta_test.txt](https://github.com/molgenis/molgenis/files/591839/sorta_test.txt) in the text area
- Select 'Match'
- Wait until the matching has completed
- Select 'Go to result' for the job with name 'sorta test'
- Select 'Match' for the first row in the 'Unmatched result' table

#### Expected
- A UI component is displayed to perform manual matching

#### Observed
```
Error! An error occurred. Please contact the administrator.
Uncaught TypeError: Cannot read property 'length' of undefined
```

**Comment by dennishendriksen** [comment]
Already fixed by @ChaoPang in https://github.com/ChaoPang/molgenis/commit/e9d8f1e8086b075d6b49b2050bb69fddf68f4e3f

**Comment by dennishendriksen** [comment]
Fixed in #6720
